src/OpenTelemetry.AutoInstrumentation.NLogTarget/OpenTelemetryTarget.cs

```diff
@@ -0,0 +1,211 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using NLog;
+using NLog.Config;
+using NLog.Targets;
+using OpenTelemetry;
+using OpenTelemetry.AutoInstrumentation;
+using OpenTelemetry.AutoInstrumentation.Configurations;
+using OpenTelemetry.Logs;
+
+namespace OpenTelemetry.AutoInstrumentation.NLogTarget;
+
+[Target("OpenTelemetryTarget")]
+public sealed class OpenTelemetryTarget : TargetWithContext
+{
+    private static readonly ConcurrentDictionary<string, object> LoggerCache = new(StringComparer.Ordinal);
+    private static LoggerProvider? _loggerProvider;
+    private static Func<string?, object?>? _getLoggerFactory;
+
+    public OpenTelemetryTarget()
+    {
+        Layout = "${message}";
+    }
+
+    [RequiredParameter]
+    public string? Endpoint { get; set; }
+
+    public string? Headers { get; set; }
+
+    public bool UseHttp { get; set; } = true;
+
+    public string? ServiceName { get; set; }
+
+    [ArrayParameter(typeof(TargetPropertyWithContext), "attribute")]
+    public IList<TargetPropertyWithContext> Attributes { get; } = new List<TargetPropertyWithContext>();
```

Comment:
Any reason for defining new `Attributes`, instead of just relying on existing from base-class ? (Called `Properties`)

Reply:
resolved in [4401757](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/4371/commits/440175748c7fc3d7ee52f84af92a7d813686ed9d)